client/test/e2e/arc/app-controller.spec.js

```diff
@@ -0,0 +1,29 @@
+var ArcViews = require('../arc/views/');
+
+var EC = protractor.ExpectedConditions;
+
+describe('app-controller-test', function() {
+  fdescribe('app-start', function() {
```

Comment:
What is `fdescribe`? If it isn't a typo, then it needs to be added to `client/test/.jshintrc` as a registered global.